src/components/Home.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import "../App.css";
+
+
+class Home extends React.Component {
+  render (){
+    return (
+      <div className='home'>
+         <body>
+           <h1>Welcome to GhibiApp</h1>
+         </body>
+      </div>
+    );
```

Comment:
```suggestion
    return (
      <div className='home'>
           <h1>Welcome to GhibiApp</h1>
      </div>
    );
```
You only need a set of `div` tags, you shouldn't use `body` tags here.